scripts/collector/postgres/sql/extensions.sql

```diff
@@ -1,4 +1,23 @@
-\o output/opdb__extensions_:VTAG.csv
-SELECT *,
-       chr(39) || :DMA_SOURCE_ID || chr(39) AS DMA_SOURCE_ID, chr(39) || :DMA_MANUAL_ID || chr(39) AS DMA_MANUAL_ID
-FROM pg_extension
+\ o output / opdb__pg_extensions_ :VTAG.csv;
```

Comment:
these spaces make the file command not work you need
\o output/opdb__pg_extensions_:VTAG.csv;